### ISSUE TITLE
Effect and Short Effect are not Reliable

### ISSUE THREAD

**Issue by GreatNovaDragon** [PokeAPI/pokeapi#1020]
There are some attacks that get constantly updated and buffed, the pure numbers get updated, but not the effect texts.

Example Fury Cutter: The Long Effect text describes an end multiplier of 16x, the short effect of "maxing out after 5 turns"
Both dont even work, cause the maxing out of power happens at 160 power, which was 16x in Gens 2-4 and the 5 turns in 5. 

Example Bind: The Effect texts claim an 1/16 of max hp damage, but that category of damage has been overall buffed to 1/8th of max hp damage. 

I did not check all moves, but I happened to see this often enough that i can't call the effect text data reliable

**Comment by Naramsim** [comment]
Good catch! We should revise all the effects. It's gonna be a tough one

**Comment by Naramsim** [comment]
As of now keep the pure numbers as a reference

**Comment by Naramsim** [comment]
I created https://github.com/PokeAPI/pokeapi/issues/1024, this can help us have a single source of truth for our numbers. 

If you are willing maybe you can help with the analysis and coding. 

Thank you!

**Comment by GreatNovaDragon** [comment]
Yeah, i am working on actually updating the move effects currently, additionally adding a translation to german for those, basing myself on the german member of the [Encyclopædiæ Pokémonis](https://www.encyclopaediae-pokemonis.org/) Pokéwiki, of which Bulbapedia is also a member. 

German Prose helps myself for my own projects, and if i do it already, i can update the english prose in one go.

**Comment by GreatNovaDragon** [comment]
Manually updating

**Comment by GreatNovaDragon** [comment]
Sorry for the Zero Progress here. The Project i had been going that depended on PokeAPI data had been archived in March. I will look into this again now.
